integration_tests/test_auth_levels.py

```diff
@@ -132,16 +124,14 @@ def test_private_without_stark_private_key(self):
             signature=signature,
         )
 
-    def test_onboard_with_private_keys(self):
+    def test_onboard_with_private_keys(self, eth_address):
```

Comment:
I don't think tests should take parameters (also see above, eth_address shouldn't be needed in get_api_keys)

Reply:
^can we remove this param, here and below?